src/nvim/drawscreen.c

```diff
@@ -1549,6 +1549,11 @@ static void win_update(win_T *wp, DecorProviders *providers)
   }
 
   bool cursorline_standout = win_cursorline_standout(wp);
+  // Make sure that the cursorline on a closed fold is redrawn
+  if (cursorline_standout) {
```

Comment:
This calculation is only needed for cursorline, not for conceal.
```suggestion
  if (wp->w_p_cul) {
```